core/src/rpc_error.rs

```diff
@@ -40,6 +44,11 @@ impl From<RpcCustomError> for Error {
                 message: "Transaction is not signed".to_string(),
                 data: None,
             },
+            RpcCustomError::BlockNotAvailable { slot } => Self {
+                code: ErrorCode::ServerError(JSON_RPC_SERVER_ERROR_4),
```

Comment:
@mvines , in https://github.com/solana-labs/solana/pull/10630 this is `JSON_RPC_SERVER_ERROR_3`, but it wasn't clear to me why. Let me know if I should revert this change.

Reply:
I think that was just a silent protest against burning server errors.  We only have 100 of them, and then the world ends